src/components/Form/index.tsx

```diff
@@ -84,11 +86,23 @@ const Form: ComponentProps = ({
         },
         watch,
         setError,
-        clearErrors
+        clearErrors,
+        getValues
     } = formApi;
     const errorFields = flat(errors);
     const layoutState = useLayout(schema, cards, layout, editors, undefined, layoutFields);
 
+    const validateCurrent = React.useCallback(() => {
+        const result = validationSchema.validate((({$original, ...values}) => values)(getValues()));
```

Comment:
I am considering such an approach as well and can't yet determine which is going to be better.

![image](https://user-images.githubusercontent.com/7376399/201926211-dc17ecf5-8020-4c16-a248-c4ad4f4c2875.png)

Reply:
The approach with validation passed as a prop of the item seems more flexible and it will amend most of the code in this MR.